src/Features/Core/Diagnostics/Analyzers/RudeEditUserDiagnosticAnalyzer.cs

```diff
@@ -24,7 +24,7 @@ public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics
         public override Task AnalyzeSyntaxAsync(Document document, Action<Diagnostic> addDiagnostic, CancellationToken cancellationToken)
         {
             // No syntax diagnostics produced by the EnC engine.  
-            return Task.FromResult(false);
+            return SpecializedTasks.EmptyTask;
```

Comment:
I've not looked at all the call sites for all of these. What are you asking be done here, switch back to equivalent behavior (e.g. .False in this case), even though it's very likely unnecessary? Or are you just pointing out there's a remote possibility this could lead to a failure but not expecting a change? Let me know what you'd like to see done and I'll follow the recommendation. I don't have a strong preference, but I do think it would be strange to change these, and I expect these were originally written this way out of convenience, not because the value of type actually mattered.